@kiva/kv-components/src/utils/index.js

```diff
@@ -1,4 +1,5 @@
 export * from './Alea';
+export { default as Alea } from './Alea';
```

Comment:
Is the `Alea` export also needed on the line above?

Reply:
When re-exporting everything from another module, it doesn't appear to be possible to do that in one line. `export * from './Alea';` will only export the named exports, not the default, and `export { default as Alea, * } from './Alea';` is not valid es syntax. Eslint reports a parsing error: `Line 1, column 27: Unexpected token *`.